GitInfo.txt

```diff
@@ -1 +1 @@
-4.1.0
\ No newline at end of file
+4.1.0
```

Comment:
I merged master into my branch to get caught up instead of 4.1.0, and this is what happened when I tried to change it back 🤷‍♂

Reply:
encoding?

Reply:
reverted via `git checkout`, seems good now